app/event-creator.tsx

```diff
@@ -1,14 +1,63 @@
 'use client'
 
-import React from "react";
+import React, { useState } from "react";
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
 
 export default function EventCreator() {
 
+   
```

Comment:
What is the purpose of these empty lines? 🤔

Reply:
I forgot about it